src/overlays/actors/ovl_En_Skjneedle/z_en_skjneedle.c

```diff
@@ -14,8 +14,10 @@ void EnSkjneedle_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnSkjneedle_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnSkjneedle_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnSkjneedle_Draw(Actor* thisx, GlobalContext* globalCtx);
+int func_80B01F6C(Actor* thisx);
```

Comment:
```suggestion
s32 func_80B01F6C(EnSkjneedle* this);
```
Does this function need a generic actor argument? It seems to be specific to only this actor.